apps/docs-app/docs/features/routing/content.md

```diff
@@ -182,3 +182,17 @@ export default class ProjectComponent {
   });
 }
 ```
+
+## Support for loading Custom Content File
+
+Analog also supports loading a custom content file from the `src/content` folder.
```

Comment:
```suggestion
By default, Analog uses the route params to build the filename for retrieving a content file from the `src/content` folder. Analog also supports using a custom filename for retrieving content from the `src/content` folder. This can be useful if, for instance, you have a custom markdown file that you want to load on a page.
```